tinkerpop3/src/main/java/io/shiftleft/overflowdb/NodeLayoutInformation.java

```diff
@@ -31,6 +31,9 @@
   /* possible edge property keys, grouped by edge label.
    * n.b. property keys are of type `HashSet` (rather than just `Set`) to ensure `.size` has constant time */
   private final Map<String, HashSet<String>> edgePropertyKeysByLabel;
+  /* maps offsetPos -> number of edge properties
+  */
+  public final int[] nPropertiesbyOffsetPos;
```

Comment:
this should be private, so that users don't change the contents

Reply:
OdbNode class needs to access it.

Reply:
```
private final int[] nPropertiesbyOffsetPos;

public final int getPropertyCountByOffsetPos(int offsetPos) { return [nPropertiesbyOffsetPos]; }
```
that way users can't shoot themselves in the foot too easily

Reply:
edit: i updated my proposal above with an (imo) better name, which also let's you drop the comment